src/vm.ts

```diff
@@ -269,6 +269,16 @@ function step(n: number = 0): void {
   runSheet.getRange(`${apColumn}${n + 2 + 1}`).setValue(newAp);
 }
 
+function run() {
```

Comment:
you can replace the `runUntilPc` or remove it

Reply:
decided to just replace runUntilPc with the run method